superset-frontend/src/components/PageHeaderWithActions/index.tsx

```diff
@@ -147,7 +147,7 @@ export const PageHeaderWithActions = ({
               <CertifiedBadge {...certificatiedBadgeProps} />
             )}
             {showFaveStar && <FaveStar {...faveStarProps} />}
-            {titlePanelAdditionalItems}
+            {/* {titlePanelAdditionalItems} */}
```

Comment:
Maybe you could delete it if we’re no longer using it

Reply:
I chose to comment it as it is not our code, is Superset's...I think it might be useful in the future to compare changes if we need to update to another version

Reply:
finally I decided to follow your comment and remove it